viewer/src/views/threejs/HtmlOverlayHelper.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import * as THREE from 'three';
+import { worldToViewport } from './worldToViewport';
+
+export class HtmlOverlayHelper {
+  private readonly htmlOverlays: Map<HTMLElement, THREE.Vector3> = new Map();
+
+  addOverlayElement(htmlElement: HTMLElement, position3D: THREE.Vector3) {
+    this.htmlOverlays.set(htmlElement, position3D);
+  }
+
+  removeOverlayElement(htmlElement: HTMLElement) {
+    this.htmlOverlays.delete(htmlElement);
+  }
+
+  updatePositions(renderer: THREE.Renderer, camera: THREE.PerspectiveCamera) {
```

Comment:
Suggest we replace 'renderer' with a `HTMLElement` and add an unit test :)

Reply:
I think passing an `HTMLElement` makes it easier to use the API wrong. I think we specifically should require the `renderer.domElement` as most other elements will not make sense to use together with `worldToViewport`. I might be wrong about this, but are there any other HTML elements that we expect users to wish to _overlay_ their HTML elements on apart from the canvas? If you want an `HTMLElement` to make the function unit-testable, I would prefer to keep the signature of this and have it call an internal function that takes an `HTMLElement` that is unit tested.

Also regarding unit testing: This is a function I would prefer to visually test, but adding a unit test is OK. I think the important outcome of this function is the visual aspect. The actual work is done by `worldToViewport`, which should be unit tested. The fact that we use `style.top` and `style.left` are the only things you can test from this function, and I think the fact that those are used (as opposed to padding, margin, etc) is bordering to implementation details. Again, not opposed to a unit test, but I would prefer a visual test :)